tedana/workflows/tedana.py

```diff
@@ -285,6 +285,9 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
     mask, masksum = utils.make_adaptive_mask(catd, mask=mask,
                                              minimum=False, getsum=True)
     LGR.debug('Retaining {}/{} samples'.format(mask.sum(), n_samp))
+    if verbose:
+        io.filewrite(mask.astype(int), op.join(out_dir, 'mask.nii'), ref_img)
```

Comment:
Is there a benefit in writing out the mask ? Either it's user-provided or effectively just the adaptive mask, no ?

Reply:
Just the regular mask? There's probably no reason to write it out. I'll drop that one.